qa/tasks/quiescer.py

```diff
@@ -173,6 +173,7 @@ def dump_ops_all_ranks(self, dump_tag):
         remote_dumps = []
 
         # begin by executing dump on all ranks
+        procs = []
```

Comment:
we already have the `remote_dumps` array, let's just put the process info there

Reply:
done